src/components/Table/SkeletonTable.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import Skeleton from 'skeletons/Skeleton';
+
+const SkeletonTableContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  & > * + * {
+    margin-top: ${({ marginBetween }) => marginBetween};
+  }
+`;
+
+const SkeletonTableHeader = styled(Skeleton.Animated)`
+  background: var(--skeleton-colors-dark);
+  border-radius: 0.5em;
+`;
+
+const SkeletonTableRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  & > * + * {
+    margin-left: ${({ marginBetween }) => marginBetween};
+  }
+  & > * {
+    flex-grow: 1;
+  }
+`;
+
+const SkeletonTableCell = styled(Skeleton.Animated)`
+  border-radius: 0.5em;
+  animation-duration: var(--skeleton-pulse-fast);
+`;
+
+const SkeletonTable = ({
+  disableAnimation,
+  width,
+  rowHeight = '40px',
+  columns = 3,
+  rows = 4,
+  marginBetween = '2px',
+}) => (
+  <SkeletonTableContainer
+    disableAnimation={disableAnimation}
+    width={width}
+    marginBetween={marginBetween}
+  >
+    <SkeletonTableHeader
+      disableAnimation={disableAnimation}
+      height={rowHeight}
+    />
+    {Array.from(Array(rows).keys()).map(rowIdx => (
```

Comment:
it seems like we can move this "array generator" to some utils and name it like: `generateArray(size)`, or `createArray(size)`

Reply:
This might be more idiomatic as: 
```
rows.map((row, rowIdx) => ( ... ));
```

Reply:
`rows` is a number though.

Reply:
ah, didn't notice that.  The chain of `Array.from(Array(rows).keys())` is pretty difficult to read.  I guess that means to use index parameter it'd have to be `Array(rows).map((_, rowIdx) => ( ... ))` which is still kinda difficult to read.